docs/general/administration/hardware-selection.md

```diff
@@ -53,6 +53,7 @@ For users with expensive power or running servers on battery power:
 
 - Intel 12th gen N series platforms
 - Apple M Series mac mini
+- Intel-Based [Framework Laptop Mainboards](https://frame.work/marketplace/mainboards)
```

Comment:
Why? We really should not endorse specific vendor made boards like this, and they are not better than a lot of other vendors.

Reply:
1. It is the only laptop manufacturer that officially supports using the mainboard outside the enclosure
2. It only uses a few watts at idle
3. It is something that our community members may have lying around
4. It is normal Intel CPUs which we already recomment

thats why I felt it was an appropriate inclusion in this section

Reply:
> It is the only laptop manufacturer that officially supports using the mainboard outside the enclosure

Why you want to use a laptop motherboard outside a laptop enclosure in the first place?

> It only uses a few watts at idle

Same for a lot of other NUCs using laptop CPUs, and you want to list all of them?

Fun fact: Using less power than an Alderlake-N processor at idle doesn't necessarily mean that the board or the SoC on that board is power-efficient enough. It just means that Alderlake-N based systems are poorly made and nobody bothers to fine-tune the power consumption.

Fun fact 2: Both RK3588 and M1 can achieve single-digit (< 10) watt power consumption during transcoding (without fancy processing like tone mapping). Being good at idling is not the only reason they are recommended. Although the real-world case is unlikely to be this ideal, but it is still way better than what the Intel Laptop chips can archive. 

> It is something that our community members may have lying around

Same for a lot of other NUCs, laptops, etc.

> It is normal Intel CPUs which we already recommend

But this is the Low-Power section, right? And we are recommending a series of CPUs that are known to be the worst of their class in power consumption, which looks absurd to me. Intel laptops after Alderlake consistently perform as the worst in battery life, which means the actual power consumption during typical loads is not something we can recommend.

Reply:
> Why you want to use a laptop motherboard outside a laptop enclosure in the first place?

For a home server, many parts of a laptop are simply not needed and the option to put it in an alternative enclosure can save space

> Being good at idling is not the only reason they are recommended.

A server is likely going to be sitting idle most of the time. Many heavy tasks are only ran once in a few months at most by default. The heaviest task a JF server is likely to see during normal use is maybe an hour of OpenCL tonemapping

Reply:
> For a home server, many parts of a laptop are simply not needed and the option to put it in an alternative enclosure can save space
> A server is likely going to be sitting idle most of the time. Many heavy tasks are only ran once in a few months at most by default. The heaviest task a JF server is likely to see during normal use is maybe an hour of OpenCL tonemapping

Then we go back to "Same for a lot of other NUCs using laptop CPUs", why this vendor and why not using an NUC but a bare board from a laptop?

The NUCs are not more power-hungry, are cheaper, and actually come with smaller motherboard. If you think the power consumption of the Intel laptop chips are good enough for recommendation, then why not recommending those but this vendor's laptop motherboard? It is almost the least reasonable choice.

Also, a lot of users won't run the server with Jellyfin only, and multitasking is one of the easiest way to make the Intel chips behave poorly in terms of power consumption because the chip is harder to get into the deep sleep state to reduce power consumption.

Reply:
Fair